profiles/base.yaml

```diff
@@ -852,5 +852,4 @@ shard_thief_password:
 forge_container:
 spell_cycle:
 
-restock_shop:
-
+restock_shop: {}
```

Comment:
❓ This is already defined in `base-empty` as `{}`. Why is it defined that way here too? I expected it to remain `restock_shop:`...